pkg/funcs/functions_test.go

```diff
@@ -0,0 +1,92 @@
+package pkg
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/cloudprivacylabs/lpg"
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
+	"github.com/cloudprivacylabs/opencypher"
+)
+
+type ageTest struct {
+	query    string
+	expected int
+}
+
+func TestAge(t *testing.T) {
+	rGetter := func(v opencypher.Value) interface{} {
+		return v.Get().(opencypher.ResultSet).Rows[0]["1"].Get()
+	}
+	ageTests := []ageTest{
+		{
+			query:    `return age(parseDate("01/01/2000", "MM/DD/YYYY"))`,
+			expected: 22,
+		},
+		{
+			query:    `return age(parseDate("01/01/1900", "MM/DD/YYYY"), parseDate("01/01/2022", "MM/DD/YYYY"))`,
+			expected: 122,
```

Comment:
Add a test case with "03/03/2001" -> "03/03/2010" and "03/03/2001" -> "03/02/2010". That's the edge case.